### ISSUE TITLE
✨ Component - `TalentNominationEventCard`

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#12313]
## ✨ Feature

Create a card component to represent a talent event

## 🕵️ Details

Make sure the card can be extended in the future to display upcoming and past events.  Those are out of scope for now, though.

## 🎨 Design

> [!NOTE]
> The image is out of scope for now. 

@JoshBeveridge 

https://www.figma.com/design/k9lJ77FgLFvpO9L9tVyzID/Talent-nomination-(Employees)?node-id=201-4719&t=35wsjXEj56gvMaGg-4

## 📸 Screenshot
> [!NOTE]
> The image is out of scope for now.

![image](https://github.com/user-attachments/assets/09eb0821-d9ac-4143-83b3-749eae6e912b)

## 🙋‍♀️ Proposed Implementation

```ts
interface TalentNominationEventCardProps {
   mode: "Current", // more modes coming soon
   communityName: string,
   title: string,
   openDate: Date,
   closeDate: Date,
   description: string,
   startUrl: string,
   learnMoreUrl?: string,
}
```

## ✅ Acceptance Criteria

- [ ] community, title, description displayed
- [ ] "start" and "learn more" navigates to the given links
- [ ] "learn more" button hidden if now URL provided
- [ ] open and close dates show in local time zone
- [ ] light mode and dark mode
- [ ] desktop and mobile layouts
- [ ] a11y: link hidden text is "Start a nomination<hidden> for {eventName}</hidden>" and "Learn more<hidden> about {eventName}</hidden>"

## 🛑 Blockers

```[tasklist]
### Blocked By
```
